src/apb/engine.py

```diff
@@ -321,6 +337,14 @@ def update_dockerfile(project):
     print('Finished writing dockerfile.')
 
 
+def load_source_dependencies(roles_path):
+    print('Trying to guess list of dependencies for APB')
+    output = subprocess.check_output("/bin/grep -R \ image: "+roles_path+"|awk '{print $3}'", stderr=subprocess.STDOUT, shell=True)
```

Comment:
I had a different definition of dependencies.  These are the images that an apb consumes. That's where I got confused.